app/core/lookups.py

```diff
@@ -68,3 +68,22 @@ def get_country_by_slug(db: Session, country_slug: str) -> Optional[Geography]:
         return None
 
     return geography
+
+
+def get_country_from_country_code(
+    db: Session, country_code: str
+) -> Optional[Geography]:
+    geography = db.query(Geography).filter_by(value=country_code).scalar()
```

Comment:
I'd probably prefer `one_or_none()` here - again for the reason that it shouldn't return more than one.